ocs_ci/framework/main.py

```diff
@@ -39,14 +39,23 @@ def init_ocsci_conf(arguments=None):
     custom_config = get_param('--ocsci-conf', arguments)
     cluster_config = get_param('--cluster-conf', arguments)
     if custom_config:
-        with open(os.path.expanduser(custom_config)) as file_stream:
-            custom_config_data = yaml.safe_load(file_stream)
-            framework.config.update(custom_config_data)
+        config_files = [cc.strip() for cc in custom_config.split(",")]
```

Comment:
I think we meant repeating --ocsci-conf , the docopt equivalent would be [ocsci-conf=conf]...

Reply:
Please see my comments about it here: https://github.com/red-hat-storage/ocs-ci/issues/700 . I don't see much additional value to have the parameter repeatable like `--ocscu-conf conf/1.yaml --ocsci-con conf/2.yaml`   . The one parameter `--ocscu-conf conf/1.yaml,conf/2.yaml` is also easy to use and does not require write `--ocscu-conf` twice.

Reply:
I can understand if its hard to fix due to our own param handling, but repeatable args are standard and better than comma separated for file names.  It also helps with auto completion on bash shell.

Reply:
yep for auto completion of files is cool thing. I changed the behavior of function to do it like that. The rest is explained in the issue.